src/screens/Documents/Documents.tsx

```diff
@@ -1,13 +1,47 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { TextRegular } from 'assets/fonts/Fonts';
 import { PageContainer } from 'screens/styles';
 import DocContainer from 'components/DocContainer/DocContainer';
+import { getDocList, getAllCases } from 'database/queries';
+import { getCurrentClient } from 'database/auth';
+import { CaseType } from 'types/types';
 
 const UploadScreen = () => {
+  const [caseTypes, setCaseTypes] = useState<CaseType[]>([]); // container headers
+  const [reqDocs, setReqDocs] = useState<string[][]>([]); // container docs
+
+  useEffect(() => {
+    async function loadClientDocs() {
+      const client = await getCurrentClient();
+      if (client !== undefined) {
+        // get all client case types
+        const cases = await getAllCases(client.id);
+        let clientCaseTypes: CaseType[] = cases.map(c => c.type);
+        clientCaseTypes = [...new Set(clientCaseTypes)];
```

Comment:
Curious - why do we need to create a set of case types before turning it into an array?

Reply:
When the clientCaseTypes would render, I would keep getting duplicate case types. I made the array into a Set to get rid of duplicate case types since Sets can't hold duplicate values.

Reply:
Makes sense! Could you add a comment message about preventing duplicate case types here?